packages/product-react/src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -0,0 +1,18 @@
+import Enzyme from 'enzyme';
+import React from 'react';
+
+import Checkbox from './Checkbox';
+
+describe('Checkbox', () => {
+  test('it renders a checkbox', () => {
+    const wrapper = Enzyme.mount(<Checkbox label="hello world" />);
+    expect(wrapper.find('.ray-checkbox').length).toBe(1);
```

Comment:
nit this test doesn't actually assert if there's a checkbox input

Reply:
done